proposed/container-meta.md

```diff
@@ -304,44 +304,32 @@ try {
 8. Implementations
 ------------------
 
-The following projects already implement the `container-interop` version of the interface and
-therefore would be willing to switch to a Container PSR as soon as it is available.
+At the time of writing, the following projects already implement and/or consume the `container-interop` version of the interface.
 
-### Projects implementing `ContainerInterface`
-
-- [Acclimate](https://github.com/jeremeamia/acclimate-container): Adapters for
-  Aura.Di, Laravel, Nette DI, Pimple, Symfony DI, ZF2 Service manager, ZF2
-  Dependency injection and any container using `ArrayAccess`
-- [Aura.DI](https://github.com/auraphp/Aura.Di) (v3+)
+- [Acclimate](https://github.com/jeremeamia/acclimate-container)
+- [Alias-Container](https://github.com/thecodingmachine/alias-container)
+- [Aura.DI](https://github.com/auraphp/Aura.Di)
+- [Behat](https://github.com/Behat/Behat)
 - [dcp-di](https://github.com/estelsmith/dcp-di)
+- [interop.silex.di](https://github.com/thecodingmachine/interop.silex.di)
+- [League Container](https://github.com/thephpleague/container)
 - [Mouf](http://mouf-php.com)
+- [mindplay/middleman](https://github.com/mindplay-dk/middleman)
 - [Njasm Container](https://github.com/njasm/container)
 - [PHP-DI](http://php-di.org)
+- [PHP-DI Invoker](https://github.com/PHP-DI/Invoker)
 - [PimpleInterop](https://github.com/moufmouf/pimple-interop)
+- [Prefixer-Container](https://github.com/thecodingmachine/prefixer-container)
+- [Prophiler](https://github.com/fabfuel/prophiler)
+- [Silly](https://github.com/mnapoli/silly)
+- [Slim](https://github.com/slimphp/Slim)
+- [Splash](http://mouf-php.com/packages/mouf/mvc.splash-common/version/8.0-dev/README.md)
 - [XStatic](https://github.com/jeremeamia/xstatic)
```

Comment:
Symfony can now be added here too!
https://github.com/symfony/symfony/pull/21265